src/modules/admin/admin.controller.ts

```diff
@@ -0,0 +1,47 @@
+import {
+	Controller,
+	Post,
+	Body,
+	UseGuards,
+	Request,
+	Get,
+	Param,
+	HttpStatus,
+} from '@nestjs/common';
+import { AdminService } from './admin.service';
+import { FieldMappingDto } from './dto';
+import { AuthGuard } from '@modules/auth/auth.guard';
+
+@Controller('admin')
+export class AdminController {
+	constructor(private readonly adminService: AdminService) {}
+
+	@Post('config')
+	@UseGuards(AuthGuard)
+	async createOrUpdatesettings(@Body() body: any, @Request() req) {
+		const mapping: FieldMappingDto = {
+			key: body.key,
+			value: body.value,
+		};
+		const result = await this.adminService.createOrUpdatesettings(
```

Comment:
@vidyaaKhandekar  API is returning the data with status OK as default, it should be based on the response returned from the service. If you do not get the data or if you got the error from the service as return value then based on this your controller should return the resposne

Reply:
@vidyaaKhandekar rename the funciton `createOrUpdatesettings` => `createOrUpdateConfig` 
Reason we are updating or creating one config at a time not multiple (Update in both controller and serivce)